alternator/stats.hh

```diff
@@ -79,6 +79,14 @@ public:
         utils::estimated_histogram batch_get_item_histogram{22}; // a histogram that covers the range 1 - 100
         utils::estimated_histogram batch_write_item_histogram{22}; // a histogram that covers the range 1 - 100
     } api_operations;
+    // Operation size metrics
+    struct {
+        // Item size statistics collected per-table per-node. Each histogram
+        // covers the range 0 - 576. The breakpoint at 400KB is there because
+        // DynamoDB doesn't support larger items. Resolves #25143.
+        // A size is the retrieved item size.
+        utils::estimated_histogram get_item_op_size_kib{32, {400}};
```

Comment:
I'm against that api with 400, but at least test it. I would have dropped it and still tested a high value